src/init.h

```diff
@@ -60,14 +61,11 @@ bool AppInitLockDataDirectory();
  */
 bool AppInitMain();
 
-/** The help message mode determines what help message to show */
-enum class HelpMessageMode {
-    BITCOIND,
-    BITCOIN_QT
-};
+/**
+ * Setup the arguments for gArgs
+ */
+void SetupArgs();
```

Comment:
nit: A more suitable name would be `SetupServerArgs`, since this is shared between bitcoind and the gui.